src/index.ts

```diff
@@ -1 +1,5 @@
-export const add = (a: number, b: number) => a + b;
```

Comment:
Version information / file header missing

Reply:
Can you explain what you mean by that?

Reply:
Related to my answer in https://github.com/leoschur/smart-range/pull/1#discussion_r933959207